tests/unit/test_image_pixels_encoder.py

```diff
@@ -57,6 +57,10 @@ def test_image_pixels_encoder(self, image_pixels):
                 '@id': 1,
                 '@type': '%s#Pixels' % OME_SCHEMA_URL,
                 'omero:methodology': 'methodology',
+                'image': {
```

Comment:
This matches what I was thinking about in previous comments. :+1: 

The one issue that I was unclear on myself is if at this level we want to make it detectable that there is no content, i.e. the equivalent of `loaded=False`. Perhaps it's enough that there are no fields other than `@id` and `@type`.

Reply:
I'm happy to add a `loaded=False` to these objects. Maybe `omero:loaded = False` to be consistent with other `omero` properties (not ome model).

Reply:
Thanks for giving that a try. Were there any locations in the code where you could make _use_ of that information? An alternate way of encoding that info would be to have the type be a `{{ type }}Ref` but then code would have to check for which of the two types was present. Summarizing (outloud) the various forms of backref encodings we've discussed here so far :

|                           | Pro                             | Con                                   |
|---------------------------|---------------------------------|---------------------------------------|
| raw integer               | Smallest footprint              | May surprise consumers; explicit test |
| implicit (i.e. no fields) | Doesn't add any new idiom       | Must check all keys                   |
| omero:loaded              | Closely matches OMERO internals | Bit verbose; difficult to deprecate   |
| TypeRef                   | Make more use of types          | Requires explicit check               |

Reply:
Under "implicit (i.e. no fields)" see https://stackoverflow.com/questions/34761970/schema-org-json-ld-reference regarding "node identifiers"

Reply:
The usage of node identifiers would allow the usage of [node reference](https://json-ld.org/spec/latest/json-ld/#dfn-node-references). This would reduce the "must check all keys" condition into "check there is only one key called `@id`". The main downside of this approach is that it requires the node identifiers to be truly unique which is not the case of `omero-marshal` at the moment (`Project:1` and `Dataset:1` have the same ID).